examples/express/src/controllers/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Request, Response } from 'express';
+import { type Request, type Response } from 'express';
```

Comment:
this was an eslint warning. If you are not seeing the warnings, maybe there is a problem with the environment setup. make sure you have the eslint extension and that you are opening the root folder instead of the example folder in the IDE.